### ISSUE TITLE
Ignore duplicate property warning when declaring background twice as fallback.

### ISSUE THREAD

**Issue by stephaniehobson** [CSSLint/csslint#546]
It's common to declare background twice when using it as a fallback for multiple background images. It'd be nice if this didn't trigger a duplicate property warning. This is similar behaviour to how background-colour doesn't trigger a duplicate property warning when providing a hex fallback for an rgba color.

Changes would also apply to the long-hand versions as well: background-repeat, background-position, etc.

This is the code that is triggering it right now:

``` css
.home {
    background-image:url("/media/redesign/img/header-background.png");
    background-repeat:repeat;
    background-color:#00539f;
    background-image:url("/media/redesign/img/header-background.png"),url("/media/redesign/img/blueprint.png"),url("/media/redesign/img/mdn-header-gradient.png");
    background-repeat:repeat, repeat, repeat-x;
    background-position:0 0,0 0,0 0;
}
```
